src/eval_mesh.py

```diff
@@ -46,6 +47,8 @@ def parse_args():
     parser.add_argument("--log", "-l", dest="logging", default="INFO", 
             choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"], help="""Set the log level. 
             Default is INFO""")
+    parser.add_argument("--dimf","-d", dest="dimf", default="1", choices=["1","2", "3"],
```

Comment:
Those changes are leftovers from testing. For the tutorial we can omit that line. 
Does anyone use this file except for the Bunny-Bloodcell example?
If you think it is very useful to extend `eval_mesh.py` to vectordata I can do that properly, otherwise I would discard the changes here.